debian/stretch/foreman/foreman.install

```diff
@@ -5,6 +5,7 @@ config/settings.yaml                     etc/foreman
 config/database.yml                      etc/foreman
 script/foreman-rake                      usr/sbin
 script/foreman-tail                      usr/sbin
+script/dynflowd                          usr/sbin
```

Comment:
why sbin? we're running the daemon as `foreman` user, so it's clearly not root only :)

Reply:
I agree but since other tools (foreman-rake, foreman-ruby, etc) were deployed there I kept the pattern rather than changing it all over. I'm happy to change it all if you 👍 it.

Reply:
`foreman-ruby` is in `bin` and so was `foreman-tasks` ;)

`f-rake` and `f-tail` are in sbin, as those really are `root` tools